tasks/utilities/task_rename_files.wdl

```diff
@@ -0,0 +1,86 @@
+version 1.0
+
+task rename_PE_files {
+  meta {
+    description: "Rename a set of Paired-End FASTQ files"
+  }
+  input {
+    File read1
+    File? read2
+    String new_filename
+    String docker = "us-docker.pkg.dev/general-theiagen/ubuntu/ubuntu:jammy-20230816"
+    Int disk_size = 100
+    Int cpu = 2
+    Int mem = 2
+  }
+  command <<<
```

Comment:
do we need to have two separate tasks for this? the way the PE  task is set up looks like we shouldn't need the SE task?